core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
> I do think that these things actually add value when the field is used more than once in the class

Having a `static final String` constant is fine (if you refer to the constant, or refer to a literal string, it doesn't matter they will both end up in the constant pool for the class), it's adding an *extra* class to hold that is unnecessary.